### ISSUE TITLE
Does this project need a license?

### ISSUE THREAD

**Issue by mankoff** [cryo-data/discuss#7]
Does the project need a license?

Two resources:
+ https://choosealicense.com/licenses/
+ https://creativecommons.org/choose/

**Comment by MartinLuethi** [comment]
Good point. Since this is less a software thing, I would tend for a Creative Commons license.
If we write extensive software, I would go for GPL or BSD.
